packages/lesswrong/lib/collections/users/newSchema.ts

```diff
@@ -856,10 +856,20 @@ const schema = {
       group: () => adminGroup,
     },
   },
+  // A mostly-legacy field. For OAuth users, includes information that was
+  // submitted with the OAuth login; for users imported from old LW, includes
+  // a link.
   profile: {
     database: {
       type: "JSONB",
     },
+    graphql: {
+      outputType: "JSON",
+      canRead: ["guests"],
+      // GraphQL resolver is provided for API back-compat (issarice's reader
+      // has it in its user fragment), but only returns an empty object.
+      resolver: (user, args, context) => ({}),
```

Comment:
Could we return something like "no longer supported"? If I'm a dev unfamiliar with our codebase exploring our graphql api, this might be confusing.

Reply:
Makes sense. Changed to that.